x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -64,6 +64,11 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables endpoint package level rbac
    */
   endpointRbacEnabled: false,
+
+  /**
+   * Enables the alert details page currently only accessible via the alert details flyout
```

Comment:
nit: it's also accessible via the table's overflow menu on the `Alert` and `Rule details` pages